examples/diagnostics_and_criticism/sampler-stats.ipynb

```diff
@@ -21,11 +21,12 @@
      "name": "stdout",
```

Comment:
<p>Here keep <code>trace.sample_stats</code>  so readers can see both variable names and attributes all at once</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/95/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>